### PR TITLE
Bn fabb62

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**
This is a very basic description because I only need it for the ID: https://github.com/BetaMasaheft/Documentation/issues/2293
Please notice that the xml:id is different than the file name - is that a problem?

### CODE REVIEW

**Comment by eu-genia** [comment]
xml:id may not be different from the filename

File: ParisBNF/abb/BNFabb62and63.xml [hunk 1]
```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:lang="en" xml:id="BNFabb62">
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb62and63.xml:3]: change ID to BNFabb62and63 or file name to BNFabb62 (whichever way but they MUST be the same)

File: ParisBNF/abb/BNFabb62and63.xml [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+
+               <msIdentifier>
+                  <repository ref="INS0303BNF"/>
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+               </msIdentifier>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb62and63.xml:31]: add `idno `and put 62 and 63 for the entire unit

File: ParisBNF/abb/BNFabb62and63.xml [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+
+<physDesc>
+
+  <objectDesc form="Codex">
+<supportDesc>
+  <extent></extent>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb62and63.xml:37]: ` <measure unit="leaf">226</measure>` ?

File: ParisBNF/abb/BNFabb62and63.xml [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+
+               <msIdentifier>
+                  <repository ref="INS0303BNF"/>
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+               </msIdentifier>
+
+<physDesc>
+
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb62and63.xml:34]: we usually have some physDesc for the entire codex here, e.g. library binding would be interesting

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Comment by MarcinKrawczuk** [comment]
Thank you, I hope it's better now.

czw., 19 sty 2023 o 11:55 Eugenia ***@***.***> napisał(a):

**Review by eu-genia — APPROVED** [review]

**Pull request merged**